book/A-git-in-other-environments/sections/visualstudiocode.asc

```diff
@@ -0,0 +1,20 @@
+=== Git in Visual Studio Code
+
+Visual Studio Code has git support built in. You will need to have git version 2.0.0 (or newer) installed.
+
+The main features are:
+
+* See the diff of the file you are editing in the gutter.
+* The Git Status Bar (lower left) shows the current branch, dirty indicators, incoming and outgoing commits.
+* You can do the most common git operations from within the editor:
+** Initialize a repository.
+** Clone a repository.
+** Create branches and tags.
+** Stage and commit changes.
+** Push/pull/sync with a remote branch.
+** Resolve merge conflicts.
+** View diffs.
+* With a extension, you can also handle GitHub Pull Requests: 
+  https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github
+
+The official documentation can be found here: https://code.visualstudio.com/Docs/editor/versioncontrol
```

Comment:
Add `[]` at the end to turn this into a clickable link.

Reply:
Whoopsie. Still need to learn to remember the ``[]``.